scripts/grab_frames.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Extracts a subset of frames from input folder into output folder
+# $1: Input folder
+# $2: Output folder
+# $3: Start frame
+# $4: Step
+# $5: End frame
+
+# Checks if file exists, copies if yes, throws waring if not.
+function check_and_copy {
+    if [ -f $1 ]
+    then
+	cp $1 $2
+    else
+ 	echo "WARNING:" $1 "not found"
+    fi
+}
+
+# Check correct number of arguments
+if [ $# -lt 5 ]
+then
+  echo "Usage: $0 INPUT_DIR OUTPUT_DIR START_FRAME STEP END_FRAME"
+  exit 1
+fi
+
+# If output directory doesn't exist, create it.
+if [ ! -d $2 ]
+then
+   mkdir $2
+fi
+
+check_and_copy $1/params_R.csv $2/
+check_and_copy $1/params_camchain.csv $2/
+
+
+for i in `seq $3 $4 $5`;
+do
+    check_and_copy $1/params_f_$i.csv $2/
+    for j in `seq 1 4`;

```

Comment:
careful about the hard coded assumption that we have 4 streams, there are also bags with only the first two streams because cameras didn't work, but probably ok for most cases